src/document/operation/insertoperation.js

```diff
@@ -58,6 +64,10 @@ CKEDITOR.define( [
 
 		_execute() {
 			this.position.parent.insertChildren( this.position.offset, this.nodeList );
+
+			return {
+				range: new Range( this.position, Position.createFromPositionAndShift( this.position, this.nodeList.length ) )
```

Comment:
In my "OT" pull request I introduced `Range.createFromPositionAndOffset` which does exactly that.